### ISSUE TITLE
High severity vulnerability

### ISSUE THREAD

**Issue by sdasda7777** [GreenDiscord/vlc-rpc#4]
When installing, I got this, do you know anything about it?

![image](https://github.com/GreenDiscord/vlc-rpc/assets/17746796/15a32c27-1518-4b4e-8d41-cb28aa48ab40)

**Comment by GreenDiscord** [comment]
Hiya! I haven't noticed this, but running the command as it says will fix anything. Last I checked these shouldn't do much, but still to be safe.

**Comment by sdasda7777** [comment]
Hey, I have no idea how to run the command, and I don't think the average user will either. I think this should be fixed in the repo, not individually by the user.

**Comment by sdasda7777** [comment]
Also this is not actually fixed, but I can't reopen it, for some reason 🙄

**Comment by sdasda7777** [comment]
Yeah, FYI, owner/collaborator closes an issue, only an owner/collaborator can reopen it, so I would advise you not to do that 🙂

**Comment by GreenDiscord** [comment]
> Hey, I have no idea how to run the command, and I don't think the average user will either. I think this should be fixed in the repo, not individually by the user.

Quoting the error: run `npm audit fix` to fix them.
